### ISSUE TITLE
`GNU bash, version 5.1.4(1)-release-(x86_64-apple-darwin18.7.0)`

### ISSUE THREAD

**Issue by brunophilipe** [brunophilipe/Cakebrew#230]
`GNU bash, version 5.1.4(1)-release-(x86_64-apple-darwin18.7.0)`

_Originally posted by @therealprof in https://github.com/brunophilipe/Cakebrew/issues/229#issuecomment-798917995_

**Comment by brunophilipe** [comment]
@therealprof Did you use `chsh` to change your default shell?

**Comment by therealprof** [comment]
Nope, that's what I use in iTerm2. The default shell is `/bin/bash` (`GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)`)

**Comment by brunophilipe** [comment]
Can you run `/bin/bash -l -c "which brew"` and check if the output is correct?

**Comment by therealprof** [comment]
seems correct:

```
# /bin/bash -l -c "which brew"
/usr/local/bin/brew
```
